### ISSUE TITLE
Try fixing MS MARCO encoding issues *before* indexing

### ISSUE THREAD

**Issue by lintool** [castorini/anserini#1212]
Simple task for next URA that comes along: https://github.com/castorini/anserini/issues/945#issuecomment-576417312

See if fixing encoding issues before indexing has any effect on MRR...

**Comment by HangCui0510** [comment]
@lintool fixing encoding issues before indexing improves MRR
<img width="490" alt="Screen Shot 2020-05-21 at 11 56 24 PM" src="https://user-images.githubusercontent.com/64120158/82629918-2466d800-9bbf-11ea-9cb5-453704e41cee.png">
<img width="197" alt="Screen Shot 2020-05-21 at 11 57 07 PM" src="https://user-images.githubusercontent.com/64120158/82629926-292b8c00-9bbf-11ea-9679-03d0b5c64363.png">
<img width="447" alt="Screen Shot 2020-05-21 at 11 56 59 PM" src="https://user-images.githubusercontent.com/64120158/82629933-2df04000-9bbf-11ea-9f1c-d7682147dbf2.png">
<img width="286" alt="Screen Shot 2020-05-22 at 12 00 19 AM" src="https://user-images.githubusercontent.com/64120158/82629949-3cd6f280-9bbf-11ea-95d5-b58670c4557c.png">

**Comment by edwinzhng** [comment]
Cool! Although not _exactly_ the same as what I had when I tried to fix the encodings manually: https://github.com/castorini/anserini/pull/947/files#diff-dfae4736bfb51a6cc825c3d227393265L88

**Comment by lintool** [comment]
hi @HangCui0510 great work!

hey @rodrigonogueira4 @ronakice - I don't think the small gain is worth us redoing everything we've done before? e.g., doc2query, running mono/duoT5, etc. right?

**Comment by ronakice** [comment]
Nah. Great that it improves MRR@10 performance, but overkill repeating everything for such an improvement. The actual impact comes from changes in R@1000 and we haven't quantified that yet.

**Comment by rodrigonogueira4** [comment]
I agree.

It would be nice to see this fix in T5 reranker, I think gains will be larger there.

**Comment by ronakice** [comment]
Another URA task ;), encoding before monoT5. I found performance degraded in duo, but didn't use this exact way of fixing it.
